hotel/site_sections/hotel_assignments.py

```diff
@@ -46,7 +46,7 @@ def goto_staffer_requests(self, id):
     @ajax
     def create_room(self, session, **params):
         params['nights'] = list(filter(bool, [params.pop(night, None) for night in c.NIGHT_NAMES]))
-        loops = int(params['count']) if params['count'] else 1
+        loops = int(params['count']) if params.get("count") else 1
```

Comment:
I believe this could at this point be shortened to:
`loops = int(params.get('count',1))`
because the `get` function lets you provide an optional 'default' value if the property it's looking for doesn't exist